.gitmodules

```diff
@@ -1,7 +1,7 @@
 [submodule "ext/llvm-project"]
 	path = ext/llvm-project
-	url = https://github.com/llvm/llvm-project
-	branch = release/17.x
+	url = https://github.com/ROCm/llvm-project
+	branch = rocm-6.4.0
```

Comment:
This is incorrect, `rocm-6.4.0` is a tag not a branch. In general `branch` field is only used when you automatically (with `git submodule update --remote`) update the submodule, it's fine to just remove it